configure

```diff
@@ -614,7 +616,8 @@ def host_arch_cc():
   for i in matchup:
     if i in k and k[i] != '0':
       rtn = matchup[i]
-      break
+      if (rtn != 's390'):
```

Comment:
not sure I follow.  I tried removing the '' around s390 and got the following error:

mhdawson@duv-aurora:~/pulls/linuxone/io.js$ ./configure
Traceback (most recent call last):
  File "./configure", line 1139, in <module>
    configure_node(output)
  File "./configure", line 682, in configure_node
    host_arch = host_arch_win() if os.name == 'nt' else host_arch_cc()
  File "./configure", line 619, in host_arch_cc
    if (rtn != s390):
NameError: global name 's390' is not defined